helpdesk_mgmt/views/helpdesk_dashboard_view.xml

```diff
@@ -1,96 +1,94 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-  <record id="helpdesk_ticket_kanban_view" model="ir.ui.view">
```

Comment:
Don't change indentation on this, as the diff is very large with no benefit

Reply:
Sorry. I'll fix this.